contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -3655,3 +3655,81 @@ END
 $$;
 GRANT EXECUTE ON PROCEDURE sys.sp_changedbowner(IN sys.sysname, IN sys.VARCHAR(5)) TO PUBLIC;
 
+CREATE OR REPLACE PROCEDURE sys.sp_procedure_params_100_managed(IN "@procedure_name" sys.sysname, 
+                                                                IN "@group_number" integer DEFAULT 1, 
```

Comment:
What is the significance of group number? If it is not relevant to Babelfish then I guess this is fine